src/Commands/Integration.ts

```diff
@@ -76,30 +76,26 @@ export class Integration implements CommandModule {
       })
       .middleware((args: Arguments) => {
         container
-          .register(IntegrationOptions, {
+          .register<IntegrationOptions>(IntegrationOptions, {
             useValue: {
-              insecure: args.insecure,
-              timeout: Number(args.timeout),
-              baseUrl: args.baseUrl,
-              user: args.user,
-              apiKey: args.password
+              // TODO: (victor.polyakov@neuralegion.con) Write correct type checking
+              timeout: +(args.timeout as number),
```

Comment:
Use `Number` instead.

Reply:
Let's do in later. We can't use just Number, because Number(string) is NaN. We should to full check param